README.md

```diff
@@ -16,13 +16,12 @@ We provided all of these schedulers because each scheduler performs differently
 - Task Type Scheduler by [Hamad Marri (TT)](https://github.com/hamadmarri/TT-CPU-Scheduler) - `linux-cachyos-tt` / `linux-tt`
 - BitMap Queue (BMQ) Alfred Chen Scheduler - `linux-cachyos-bmq`
 - Priority and Deadline based Skiplist multiple queue scheduler (PDS) - Alfred Chen Scheduler » `linux-cachyos-pds`
-- CacULE and CacULE-RDB created from Hamad Marri, maintained by CachyOS - `linux-cachyos-cacule` ATTENTION: Not supported after 6.1
```

Comment:
I think we should not delete this line, but just underline that it is no longer supported

Reply:
We can put a link to:
https://github.com/ptr1337/linux-cacule maybe.

Reply:
* CacULE and CacULE-RDB created by Hamad Marri, supported by CachyOS in the past as - inux-cachyos-cacule
  **ATTENTION:** Not supported after version 6.1. If you still want to use it, you can get it from the archive repository - https://github.com/ptr1337/linux-cacule.

Reply:
yep

Reply:
so, let's just commit this?